com/win32com/client/connect.py

```diff
@@ -24,9 +24,9 @@ def __del__(self):
     def _wrap(self, obj):
         useDispatcher = None
         if self.debug:
-            from win32com.server import dispatcher
-
-            useDispatcher = dispatcher.DefaultDebugDispatcher
+            from win32com.server.dispatcher import (
```

Comment:
Does it matter much to delay this import to if and when wrap is called in debug? Otherwise I'd move the import to the top to further simplify like the rest of the changed code